manifests/metallb-frr.yaml

```diff
@@ -1,3 +1,411 @@
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
@fedepaol can you move CRDs definition and controller/speaker service account config into a separate file(s) ? 
This way operator would just refer to these file(s) and deploy it.

Reply:
My idea is, we switch to kustomize (where we can have a folder with CRDs only),  and THEN we generate the -frr and vanilla manifest from there